docs/faq/data-loading.md

```diff
@@ -0,0 +1,60 @@
+# Data Loading
+
+### **Why don’t I see any data in my destination yet?**
+
+It can take a while for Airbyte to load data into your destination. Some sources have restrictive API limits which constrain how much data we can sync in a given time. Large amounts of data in your source can also make the initial sync take longer. You can check your sync status in your connection detail page that you can access through the destination detail page or the source one.
+
+### **What happens if a sync fails?**
+
+You do not lose data when a sync fails, but no data is added or updated in your destination. 
+
+Airbyte will make 3 separate attempts automatically. You can see and export the logs for those sync attempts from within the connection detail page, which you can access through the Source or Destination detail page. 
+
+In the future, we might send you an email notification \(optional, of course\) with an additional option to create a GitHub issue with the logs. We’re still thinking about it, but the purpose would be to help the community and the Airbyte team fix the issue as soon as possible, if it’s an issue with the connector. 
+
+In the meantime, here is what you can do now:
+
+* File a GitHub issue: go [here](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=type%2Fbug&template=bug-report.md&title=) and file an issue with the detailed logs copied in the issue’s description. The team will be notified about your issue and will update it for any progress or comment on it.  
+* Fix the issue yourself: with open source, you don’t need to wait for anybody to fix your issue if it is important to you. In that case, just fork the [GitHub project](http://github.com/airbytehq/airbyte) and fix the connector you need fixed. If you’re okay with contributing your fix to the community, please then create a pull request. Don’t hesitate to ping the team on [Slack](https://slack.airbyte.io), so we can check your PR as soon as possible. But you do NOT need to wait for the PR to be approved to benefit from your own fix. Put your connector in a new folder, and add your connector directly through our UI by clicking on + New connector in the Admin section. This way, you will be able to use your connector as a separate one from the connector available to the community. 
```

Comment:
```suggestion
* Fix the issue yourself: Airbyte is open source so you don’t need to wait for anybody to fix your issue if it is important to you. To do so, just fork the [GitHub project](http://github.com/airbytehq/airbyte) and fix the piece of code that need fixing. If you’re okay with contributing your fix to the community, you can submit a pull request. We will review it ASAP. 
* Ask on Slack: don’t hesitate to ping the team on [Slack](https://slack.airbyte.io).
```

Reply:
For the fix the issue yourself, I would add a link to the guide to do so instead of trying to explain what to do in one line